test/steps/coworker_steps.go

```diff
@@ -0,0 +1,33 @@
+package steps
+
+import (
+	"fmt"
+
+	"github.com/cucumber/godog"
+)
+
+// CoworkerSteps defines Gherkin step implementations around a coworker.
+func CoworkerSteps(suite *godog.Suite, fs *FeatureState) {
+	suite.Step(`^my coworker fetches updates$`, func() error {
+		_, err := fs.activeScenarioState.gitEnvironment.CoworkerRepo.Shell.Run("git", "fetch")
+		return err
+	})
+
+	suite.Step(`^my coworker sets the parent branch of "([^"]*)" as "([^"]*)"$`, func(childBranch, parentBranch string) error {
+		_, err := fs.activeScenarioState.gitEnvironment.CoworkerRepo.Shell.Run("git", "config", "git-town-branch."+childBranch+".parent", parentBranch)
```

Comment:
Such Git Town configuration operations are now centralized. You can use the [Configuration.SetParentBranch](https://github.com/git-town/git-town/blob/master/src/git/config.go#L411-L413) method here:

```go
_, err := fs.activeScenarioState.gitEnvironment.CoworkerRepo.Configuration(false).SetParentBranch(childBranch, parentBranch)